lang/python/py/patches/001-do-not-use-setuptools.patch

```diff
@@ -0,0 +1,19 @@
+Index: py-1.8.0/setup.py
+===================================================================
+--- py-1.8.0.orig/setup.py
++++ py-1.8.0/setup.py
```

Comment:
Would you please remove it the exact version? If there is a new release, you will need to rebase the patch. There's a chance that you will don't need to do it for each version. In that case, the refresh should be enough.

Reply:
Removed